content/docs/user-guide/components/reference/gameplay/fly-camera-input.md

```diff
@@ -0,0 +1,37 @@
+---
+linkTitle: Fly Camera Input
+title: Fly Camera Input Component
+description: Use the Fly Camera Input component to add quickly add a controllable camera to your Open 3D Engine (O3DE) level.
+---
+
+Use the **Fly Camera Input** component to quickly add a controllable camera to your level. The Fly Camera Input component receives inputs from a mouse and keyboard, gamepad, or touch device.  You can control the forward, backward, and lateral movement as well as the look direction of a camera component that is attached to the same entity as the Fly Camera Input component.  Mouse X and Y-axis movements control look direction. The keyboard keys `W`, `A`, `S`, and `D` control forward, left, backward, and right motion respectively.
+
+## Provider
+
+Atom Bridge Gem
```

Comment:
Atom Bridge and Atom O3DE Integration are listed as different gems in the project manager.  I didn't link to Bridge because it doesn't have a gem reference page.  I'm not sure of the function of either though, so let me know if a link to O3DE integration is appropriate and I'll add it.

Reply:
Since the Atom O3DE Integration Gem contains / is a parent of the Atom Bridge Gem, and since we don't have separate docs for the Bridge Gem, I'd say this is an acceptable link to use.